.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+# Description
+
+Please include a summary of the change and specify which issue is being addressed. Additionally, provide relevant motivation and context.
+
+Fixes # (issue number)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Documentation (typos, code examples, or any documentation updates)
+- [ ] Bug fix (a non-breaking change that resolves an issue)
+- [ ] New feature (a non-breaking change that adds functionality)
+- [ ] Breaking change (a fix or feature that would disrupt existing functionality)
+- [ ] This change requires a documentation update
+
+## Checklist
+
+- [ ] My code adheres to the style guidelines of this project (`scripts/lint.sh` shows no errors)
```

Comment:
what about running tests and static analysis? there are scripts for that too

Reply:
Few lines later there is this following line

`Both new and existing unit tests pass successfully on my local environment with my changes`

I will update it to run tests using scripts and also will add a line to run static analysis.

Reply:
I know, but it needs to be consistent. Every step must have a name of the script to be used.